gems/sorbet-runtime/lib/types/private/sealed.rb

```diff
@@ -7,6 +7,11 @@ def inherited(other)
       super
       this_line = Kernel.caller.find {|line| !line.match(/in `inherited'$/)}
       T::Private::Sealed.validate_inheritance(this_line, self, 'inherited')
+      @sorbet_sealed_module_all_subclasses << other
+    end
+
+    def sealed_subclasses
+      @sorbet_sealed_module_all_subclasses
```

Comment:
Do you think it's worth trying to freeze this set at some point? Like maybe freeze the set on first usage, so that it would break if you tried to `include` / `inherit` the parent class after accessing `sealed_subclasses` once.